pkg/pullsync/metrics.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package pullsync
+
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metrics struct {
+	OfferCounter    prometheus.Counter // number of chunks offered
+	WantCounter     prometheus.Counter // number of chunks wanted
+	DeliveryCounter prometheus.Counter // number of chunk deliveries
+	DbOpsCounter    prometheus.Counter // number of db ops
+}
+
+func newMetrics() metrics {
+	subsystem := "pullsync"
+
+	return metrics{
+		OfferCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: m.Namespace,
+			Subsystem: subsystem,
+			Name:      "chunks_offered",
+			Help:      "Total chunks offered.",
+		}),
```

Comment:
if i understand this correctly (and please correct me if i don't) this is about the chunks that are offered to the running node, no?

if so, can we make this more explicit in the help text?

i initially thought this was about chunks offered to _other_ nodes

Reply:
it coincides with the terms used in the pull sync protocol

Reply:
fair enough